### ISSUE TITLE
Setup ConvNet that overfits on the training data (we regularize afterwards)

### ISSUE THREAD

**Issue by gabrieldernbach** [gabrieldernbach/acoustic_train_classification#31]
If a network is not overfitting on the training data it might be too small for the problem at hand in the first place.

**Comment by gabrieldernbach** [comment]
Tested Resnt101, converged on 85 % accuracy on balance resampled training set.

This suggests that either one of the following
a) the representation induces a significant class overlap
b) the labels are noisy
c) the model not big enough (highly unlikely, this is why we took 101 layers in the first place)
